torch_geometric/loader/utils.py

```diff
@@ -156,3 +157,112 @@ def filter_custom_store(
         data[attr.group_name][attr.attr_name] = tensors[i]
 
     return data
+
+
+# Input Utilities #############################################################
+
+
+def get_input_nodes(
```

Comment:
IMO, makes more sense to move this to `node_loader.py`

Reply:
I'm not totally sure, since this is generic enough that if someone wanted to write their own loader, it may also use `get_input_nodes`.

Reply:
Ok :)